static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,166 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Other projects can then share these artifacts by downloading (`dvc get`) or
+importing (`dvc import`) them for use in different data processes – and these
+don't even have to be _DVC projects_, as `dvc get` works anywhere in your
+system.
+
+The advantages of using a DVC **data registry** project are:
+
+- [Versioning](/doc/use-cases/data-and-model-files-versioning): Any version of
+  the data or ML modes tracked by a DVC registry can be used in other projects
+  at any time.
+- Data as code: Version straightforward dataset directory structures without
```

Comment:
Data as Code concept includes is more about `Lifecycle management` below. Just a fancy name. To some extent `Lifecycle management` includes versioning. So, it can be just one point for all three - `Data as Code` which takes description from lifecycle mngt and add a link to versioning. May be with some wording changes.

Reply:
I merged "Data as code" and "Lifecycle management" in e37e4bac. Let me try to merge them all...

Reply:
OK I merged them all as suggested in ba2e28e3. The first bullet is kind of long now though... Please take a look feel free to resolve this conversation if it looks good.